src/Symfony/Component/Console/Tests/Command/ListCommandTest.php

```diff
@@ -80,8 +80,7 @@ public function testExecuteListsCommandsOrder()
   -h, --help            Display help for the given command. When no command is given display help for the list command
   -q, --quiet           Do not output any message
   -V, --version         Display this application version
-      --ansi            Force ANSI output
-      --no-ansi         Disable ANSI output
+      --[no-]ansi       Force [or Disable] ANSI output
```

Comment:
Should be `--ansi, --no-ansi  Force [or disable] ANSI output`. The text describer in Symfony is [docopt](http://docopt.org/) compatible.